apps/site/src/components/service-message-list.tsx

```diff
@@ -1,25 +1,25 @@
 import {
+  JsonObject,
   ServiceDefinition,
   ServiceMessageDefinition,
 } from "@blockprotocol/core";
 import { Box, Typography } from "@mui/material";
 import { Fragment, FunctionComponent } from "react";
 
-import { JsonSchema } from "../lib/json-schema";
```

Comment:
I removed the `json-schema` file – we should settle on a canonical JsonSchema type and use it where appropriate. This doesn't really matter for now.